### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by murphy78ers**
Create README for documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# repo2
```
Comment (Tosin77) [inline README.md]: add a little bit of documentation

Comment (Tosin77) [reply README.md]: add a little bit of documentation

**Review by Tosin77 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (murphy78ers) [reply README.md]: documentation added

Comment (Tosin77) [reply README.md]: add a littyle bit more

**Review by Tosin77 — CHANGES_REQUESTED** [review]
i want you to add a little bit more documentation

File: README.md (hunk 1 above)
Comment (murphy78ers) [reply README.md]: fixed the document start words

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# repo from murphy update
+documentatuon
+now this is the second time i am going to add documentation to this file
```
Comment (Tosin77) [inline README.md]: make the word file bold

**Review by Tosin77 — CHANGES_REQUESTED** [review]
add another  python file to this project

File: README.md (hunk 2 above)
Comment (murphy78ers) [reply README.md]: the word file is now in capital as requested

**Comment by murphy78ers** [comment]
python file was also added to the project as requested

**Review by murphy78ers — COMMENTED** [review]
i review and corrected them all

**Review by Tosin77 — CHANGES_REQUESTED** [review]
reviewed and it ok to merge PR

**Review by Tosin77 — APPROVED** [review]

**Pull request merged**